### PR TITLE
Prepare Happa for k8s 1.25

### PR DESCRIPTION
**Pull request by allanger**
### What does this PR do?

I'm removing PSP and replacing it with PSS, so the chart is compatible with k8s 1.25

### What is the effect of this change to users?

Users will be able to install it on k8s 1.25

### What is needed from the reviewers?

Reviewers should check that I've set PSS as we expect them to be set

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@4.35.6
+  architect: giantswarm/architect@4.38.0
```
Comment (marians) [inline .circleci/config.yml:3]:
```suggestion
  architect: giantswarm/architect@5.0.0
```

File: helm/happa/values.yaml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 name: happa
 namespace: giantswarm
+
+registry:
+  domain: gsoci.azurecr.io
```
Comment (marians) [inline helm/happa/values.yaml:5]: This is redundant

```suggestion
```

**Comment by marians** [comment]
CI fails with this error

```
helm/happa/values.yaml
  101:1     error    duplication of key "registry" in mapping  (key-duplicates)
```

because `.registry` was in fact already set in the values.yaml file.

**Comment by allanger** [comment]
> CI fails with this error
> 
> ```
> helm/happa/values.yaml
>   101:1     error    duplication of key "registry" in mapping  (key-duplicates)
> ```
> 
> because `.registry` was in fact already set in the values.yaml file.

Yes, I see, but before it was failing with another error, probably because of an outdated archorb version

**Review by uvegla — APPROVED** [review]

**Pull request merged**